tekton/resources/ci/github-template.yaml

```
@@ -48,25 +56,26 @@ spec:
             #!/bin/sh
             mkdir -p $(resources.outputs.pr.path)
             cp -r $(resources.inputs.pr.path)/* $(resources.outputs.pr.path)/
-        - name: setup-comment
+        - name: update-check-status
           image: python:3-alpine
           script: |
             #!/usr/bin/env python
             import json
             import random
 
             check_status = "$(tt.params.checkStatus)"
+            if "$(params.parentPipelineRunName)" == "":
+              resource_path = "taskruns/$(params.taskRunName)"
+            else:
+              resource_path = "pipelineruns/$(params.parentPipelineRunName)?pipelineTask=$(params.parentPipelineRunTaskName)"
 
-            if (check_status == "complete"):
-              logs_url = 'http://logs.dogfooding.tekton.dev/?buildid=$(tt.params.buildUUID)&namespace=tektonci'
-              if ("$(tt.params.checkResult)" == "True"):
-                check_status = "success"
-              elif ("$(tt.params.checkResult)" == "False"):
-                check_status = "failure"
-              else:
-                check_status = "error"
+            if ("$(tt.params.checkResult)" == "True"):
+              check_status = "success"
+            elif ("$(tt.params.checkResult)" == "False"):
+              check_status = "failure"
             else:
-              logs_url = 'https://dashboard.dogfooding.tekton.dev/#/namespaces/tektonci/taskruns/$(tt.params.taskRunName)'
+              check_status = "error"
+            logs_url = 'https://dashboard.dogfooding.tekton.dev/#/namespaces/#/namespaces/$(params.taskRunNamespace)/' + resource_path
```

Looks like an extra `/#/namespaces` in the URL that should be removed:

```suggestion
            logs_url = 'https://dashboard.dogfooding.tekton.dev/#/namespaces/$(params.taskRunNamespace)/' + resource_path
```

---

Oh, nice catch, thank you!